src/utils/components/ErrorWindow.vue

```diff
@@ -40,7 +42,9 @@ const emit = defineEmits(['close'])
             class="card-header d-flex align-items-center justify-content-sm-end"
             data-cy="window-header"
         >
-            <span v-if="title" class="me-auto text-truncate">{{ i18n.t(title) }}</span>
+            <span v-if="title" class="me-auto text-truncate"
+                >{{ i18n.t(title) }}({{ errorCount }})</span
```

Comment:
I think it would make sense to show the error count only if there are more than one error